src/components/Pages/PlaylistCollection/widgets/Playlist/Playlist.js

```diff
@@ -16,6 +16,30 @@ const PlaylistWidget = ({ playlist, selected = 0, link, t }) => {
   const location = useLocation();
 
   const { collection, items, name } = playlist;
+  const { content_type, ccuNames } = collection;
+
+  const getCollectionPartNumber = unitId => {
+    // put items with not valid collection part numbers at the end
```

Comment:
We should try to keep the original order for collections with no part number.
What do you think?

Reply:
the task was to make that order, the problem is that not all the items have part number, although being a collection

Reply:
fixed the sorting, try now